user-guide/Basic_Functionality/Alarms/Working_with_alarms/Advanced_analytics_features/Behavioral_anomaly_detection.md

```diff
@@ -20,3 +20,6 @@ Please note the following regarding suggestion events:
 - Suggestion events generated to indicate a behavioral anomaly are automatically cleared 2 hours after their creation time or their last update time. From DataMiner 10.2.11/10.3.0 onwards, they are also cleared in case a new behavioral change is detected that ends the previous anomalous behavioral change. For example, when an alarm was created for an anomalous level increase at 1 PM, and a behavioral change point is detected at 2 PM when the level drops again, then the alarm created at 1 PM will be closed at 2 PM.
 
 - Prior to DataMiner 10.2.11/10.3.0, suggestion events are created for all anomalous behavioral changes that do not have alarm monitoring enabled. From DataMiner 10.2.11/10.3.0 onwards, they are only created for the most significant changes. There is also a maximum of 500 suggestion events related to behavioral anomaly detection at the same time.
+
+> [!NOTE]
+> From DataMiner 10.3.3/10.4.0 onwards, anomaly detection is not available for discrete parameters.
```

Comment:
Personnaly, I feel like it is sufficient to mention that anomaly detection is not available for discrete parameters in the "Working_with_behavioral_anomaly_detection.md" file only. In fact, behavioral anomaly detection can also not be applied on string parameters or other non-numeric parameters, and cannot be applied on a partial table parameter, but this is not mentioned here. It is bit weird to have it only mentioned for the discrete type then?